code/datums/diseases/kingstons.dm

```diff
@@ -43,11 +43,12 @@
 					affected_mob.visible_message("<span class='danger'>[affected_mob]'s form contorts into something more feline!</span>", \
 													"<span class='userdanger'>YOU TURN INTO A TAJARAN!</span>")
 					var/mob/living/carbon/human/catface = affected_mob
-					catface.set_species(/datum/species/tajaran)
+					catface.set_species(/datum/species/tajaran, retain_damage = TRUE)
 
 
-/datum/disease/kingstons/advanced
+/datum/disease/kingstons_advanced //this used to be directly a subtype of kingstons, which sounds nice, but it ment that it would *turn you into a tarjaran always and have normal kingstons stage act* Don't make virusus subtypes unless the base virus does nothing.
```

Comment:
Comment nitpick: this a bit misleading. It should say something like "Don't make viruses subtypes unless the base virus is written in a way to handle these subtypes." 

For example, the transformation viruses (and some others) are set up to handle subtypes. They all run the parent stage act, but with some code of their own.